pkg/kubelet/kubelet_pods.go

```diff
@@ -435,6 +435,21 @@ func (kl *Kubelet) GetPodCgroupParent(pod *v1.Pod) string {
 	return cgroupParent
 }
 
+// GetHostNameForKernel gets hostname value to set in the hostname field (the nodename field of struct utsname) of the pod.
```

Comment:
GetNodenameForKernel is more explicit here.

Reply:
thanks, changing.